users/views.py

```diff
@@ -1,19 +1,25 @@
 from django.contrib.auth import views as auth_views, get_user_model
 from django.contrib.auth.forms import PasswordChangeForm
-from django.contrib.auth.mixins import LoginRequiredMixin, UserPassesTestMixin
+from django.contrib.auth.mixins import LoginRequiredMixin, \
+    PermissionRequiredMixin, UserPassesTestMixin
 from django.shortcuts import render, redirect
 from django.urls import reverse
-from django.views.generic import CreateView, UpdateView
-
-from .forms import UserRegisterForm, UserUpdateForm
+from django.views.generic import CreateView, UpdateView, ListView, DeleteView
+from django.contrib.auth.models import Group
+from django.urls import reverse_lazy
+from .forms import UserRegisterForm, UserUpdateForm, GroupForm
```

Comment:
You should have blank line before local imports see [pep8](https://www.python.org/dev/peps/pep-0008/#imports)